roxie/ccd/ccdserver.cpp

```diff
@@ -4082,6 +4084,12 @@ class CRemoteResultAdaptor : implements IEngineRowStream, implements IFinalRoxie
                 IRoxieQueryPacket *i = p.queryPacket();
                 if (i)
                 {
+                    if (timeout)
+                    {
+                        if (!i->resendNeeded(timeout))
+                            continue;
+                        activity.queryLogCtx().CTXLOG("Input has not been acknowledged for %u ms - retry required?", timeout);
```

Comment:
could this ever get logged too often ?

Reply:
If a network is very sick, or a timeout is too low... A general mechanism for spotting excessive logging and deduping might be good, or we can roll our own for each instance I suppose...